CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+to offer, add a comment. Otherwise, read on!
+
+#### How Do I Submit a (Good) Bug Report?
+
+Bugs are tracked as [GitHub issues](https://help.github.com/articles/about-issues/).
+When adding a new issue, explain the problem and include additional details to
+help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps for reproducing the problem** using as many details
+as possible.
+* **Provide specific examples to demonstrate the steps**. Include links to files,
+GitHub projects, or copy/pasteable snippets in which you use in those examples.
+If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the behavior you observed after following the steps** and point out
+what exactly is the problem with that behavior.
+* **Explain the behavior you expected to see instead and why.**
+
+### Suggesting Enhancements
+
+Enhancements include everything from suggestions to completely new features or
+minor improvements to existing functionality.
+
+Before proposing an enhancement, please perform a quick search to see if it has
+already been proposed. If your enhancement has already been proposed but you have
+new ideas to offer, add a comment. Otherwise, read on!
+
+#### How Do I Submit a (Good) Enhancement Suggestion?
+
+Enhancements are tracked as [GitHub issues](https://help.github.com/articles/about-issues/).
+When adding a new proposal, collect your reasons for why this enhancement should
+be implemented and do the following:
+
+* **Use a clear and descriptive title** for the issue to identify the proposal.
+* **Provide a step-by-step description of the suggested enhancement** in as many
+details as possible.
+* **Describe the current behavior** and/or **explain which behavior you want to
+see instead**
+* **Explain why this enhancement would be useful**
+
+### Pull Requests
+
+Issues that we need help on will be tagged as `help-wanted` and listed under "To Do" in [Projects](https://github.com/HackIllinois/api-2017/projects).
+If you're interested in tackling an open issue, leave a comment and let us know your
+intent.
+
+New contributors will want to look for issues that are also tagged as
+`beginner`. These are issues that we think are well-suited for individuals that
+are not too well-acquainted with the codebase.
+
+After you've made the appropriate changes, open a new [GitHub pull request](https://help.github.com/articles/about-pull-requests/).
+Be sure to review the guidelines below!
+
+#### How Do I Submit a (Good) Pull Request?
+
+All changes must be reviewed before being merged into the codebase, and some
+additional changes may be requested before the pull request is accepted. In order
+to make this process as easy as possible, be sure to do the following:
+
+* **Select your merge target** to be `staging` or lower, never `master`
```

Comment:
Yeah that's what I meant. Do you think I should just explicitly say "or branches that have been branched off of `staging`" instead?